packages/engine/Specs/Scene/Model/ModelSpec.js

```diff
@@ -3732,6 +3732,7 @@ describe(
         scene.verticalExaggeration = 2.0;
         scene.renderForSpecs();
         expect(resetDrawCommands).toHaveBeenCalled();
+        scene.verticalExaggeration = 1.0;
```

Comment:
Should this be set in `afterEach` just in case?

Reply:
Good point! I added this, to prevent future problems if someone adds another exaggeration-related spec.